optuna/study.py

```diff
@@ -149,6 +149,15 @@ def trials(self) -> List[FrozenTrial]:
 
         return self.get_trials(deepcopy=True, states=None)
 
+    def _is_multi_objective(self) -> bool:
+        """Return True if the study has multi objectives.
+
+        Returns:
+            A boolean value indicates if the list `study.directions` has more than 1 element or not.
+        """
+
+        return True if len(self.directions) > 1 else False
```

Comment:
```suggestion
        return len(self.directions) > 1
```

operator `>` returns a boolean value. So we can use the result directly.